tasks/testwasher.py

```diff
@@ -88,3 +101,37 @@ def process_module_results(self, module_results: List[ModuleTestResult]):
             print(failed_tests_string)
 
         return should_succeed
+
+    def is_known_flaky_test(self, failing_test, known_flaky_tests, known_flaky_tests_parents):
+        """
+        Check if a test is known to be flaky
+        The method should be called with the following arguments:
+        - failing_test: the test that is failing
+        - known_flaky_tests: the set of tests that are known to be flaky
+        - known_flaky_tests_parents: the set of tests that are known to be flaky with their parents
+        If a test is a parent of a test that is known to be flaky, the test should be considered flaky
+        For example:
+        - if TestEKSSuite/TestCPU is known to be flaky, TestEKSSuite/TestCPU/TestCPUUtilization should be considered flaky
+        - if TestEKSSuite/TestCPU is known to be flaky, TestEKSSuite should be considered flaky
+        - if TestEKSSuite/TestCPU is known to be flaky, TestEKSSuite/TestMemory should not be considered flaky
+        """
+
+        failing_test_parents = self.get_tests_parents([failing_test])
+
+        if any(parent in known_flaky_tests for parent in failing_test_parents):
+            return True
+
+        return failing_test in known_flaky_tests_parents
+
+    def get_tests_parents(self, test_name_list):
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
    def get_tests_family(self, test_name_list):
```
I prefer the `family` because it includes the test itself, compared to `parents` that might mean the returned value do no include the test itself